modules/Core/src/main/java/org/terasology/rendering/nui/layers/hud/InventoryHud.java

```diff
@@ -69,4 +74,21 @@ public Boolean get() {
             return component != null && component.slot == slot;
         }
     }
+
+    public class CurrentSlotItem extends ReadOnlyBinding<String> {
```

Comment:
I'm getting a warning from IntelliJ here about making the class final. Since it is only used here anyway maybe `private final` instead of `public` ?